torch/nn/attention/__init__.py

```diff
@@ -34,7 +34,15 @@
 SDPBackend = SDPBackend
 r"""An enum-like class that contains the different backends for scaled dot product attention.
     This backend class is designed to be used with the sdpa_kernel context manager.
-    See :func: torch.nn.attention.sdpa_kernel for more details.
+
+    The following Enums are available:
+        - ERROR: An error occurred when trying to determine the backend.
+        - MATH: The math backend for scaled dot product attention.
+        - FLASH_ATTENTION: The flash attention backend for scaled dot product attention.
+        - EFFICIENT_ATTENTION: The efficient attention backend for scaled dot product attention.
+        - CUDNN_ATTENTION: The cuDNN backend for scaled dot product attention.
+
+    See :func:`torch.nn.attention.sdpa_kernel` for more details.
 
     ... warning:: This class is in beta and subject to change.
```

Comment:
nit: Rendering seems broken for the warning :/

```suggestion
    .. warning:: This class is in beta and subject to change.
```

Reply:
Awesome, thanks for taking a look I will update